src/Compilers/Core/Desktop/SimpleAnalyzerAssemblyLoader.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
```

Comment:
This type can't exist in it's current form in this namespace.  It uses `Assembly.LoadFrom` which is a banned API in coreclr.  We need to abstract that away somehow.